kolibri/plugins/learn/assets/src/vue/content-render.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
`kolibri/plugins/learn/assets/src/vue/content-render.vue`:

* At one point we discussed this being in core. I think it's fine if it's not, but I was curious whether this was intentional?
* should we call it `content-renderer`?